Stimuli/Panda.py

```diff
@@ -137,10 +137,11 @@ def setup(self):
         self.render.setLight(self.ambientLightNP)
 
     def prepare(self, curr_cond, stim_period=''):
-        self.flag_no_stim = stim_period not in curr_cond
```

Comment:
I guess you made that change because if stim_period was empty this failed? Otherwise I can't see why you needed that change

Reply:
I think this flag is wrong and I should remove it.

There are cases like in the MatchPort that using empty period and with the flag it doesn't work because I use the flag in line 191 on pandas.py where I do not allow to run it because flag_no_stim is True. 

The only use case that I find meaningful is when the period is not empty stim_period!=' ' or stim_period not in curr_cond and in that case it should raise an exception not use the return.

Reply:
Now I remember why I added the flag_no_stim because I wanted to have stimulus at delay period and I add this line:
https://github.com/ef-lab/PyMouse/blob/c86b6d15ed5f5516e75483b79400a380365871db/Experiments/MatchToSample.py#L130

but on the most of the experiments we do not have stimulus at the delay period. So I am checking if delay is in conditions otherwise I overpass functions prepare,start and stop with the flag_no_stim. I did this in order to avoid making a new experiment just for adding stimulus on Delay period. 

I think is useful to have a behavior like this but the solution with the flag is not really good.